src/views/resolve.ts

```diff
@@ -33,3 +33,38 @@ export function resolveIds(fromIDType: IDType, ids: Range | number[], toIDType:
   // assume mappable
   return fromIDType.mapToFirstName(ids, target);
 }
+
+export function resolveNames(fromIDType: IDType, ids: Range | number[], toIDType: IDType | string = null): Promise<string[]> {
```

Comment:
Please add some JSDoc the new (and maybe also the existing) functions

Reply:
Documentation added to all functions